react-web/src/components/molecules/input-text/index.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react'
+
+import classNames from 'classnames'
+
+import { IInputProps } from 'components/molecules/input-text/type'
+
+import Eth from 'app/core/resources/eth-icon.svg'
+
+import { Label } from '../../atoms/typography/label'
+import styles from './styles.module.scss'
+
+export enum InputLabel {
+  sending = 'Sending',
+  receive = 'Receive',
+}
+
+export enum CurrencyLabel {
+  eth = 'ETH',
+  weth = 'WETH',
+}
+
+export interface IInputTextProps extends IInputProps {
+  htmlType?: string
+  currency: CurrencyLabel
+  label: InputLabel
+}
+
+const InputText = React.forwardRef<HTMLInputElement, IInputTextProps>(
+  (
+    {
+      name,
+      onChange,
+      htmlType = 'number',
+      className,
+      id,
+      currency,
+      label,
+      ...restProps
+    },
+    ref
+  ): JSX.Element => {
+    const [hasBalanceInfo, setHasBalanceInfo] = useState(false)
+
+    return (
+      <div style={{ maxWidth: 600, margin: 32 }}>
```

Comment:
Does this part need to be `style` or could you change it to `className` and add the styles to the file?